src/vulnpy/trigger/__init__.py

```diff
@@ -1,3 +1,14 @@
 from vulnpy.trigger.util import create_trigger_map, get_trigger  # noqa: F401
 
 TRIGGER_MAP = create_trigger_map()
+
+
+# This test data assumes that all triggers for the vulnerability accept the same
+# user input. However, this isn't always true, as in the case of deserialization
+# pickle and yaml. However, for test purposes, this is fine.
+DATA = {
```

Comment:
Because I'm actually hoping we will be able to use this at some point for the templates 🤞  ALso `from vulnpy.trigger import DATA` looks pretty great! but anyway if you realy want to I can move it.

Reply:
if you think it might be useful for templating that would be awesome - I think I see where you're going with this. With that in mind I like leaving it right here

Reply:
pie in the sky goal would be to use these for where the templates have `placeholder=...` but maybe that's just dreaming.

Reply:
That's what I was thinking! Although it would be kind of nice if we could copy the attack, since as soon as you click on the input field it disappears. Or maybe we could even add a button for "submit premade attack" or something. Anyway, this leaves us open to lots of future option